browser/components/preferences/in-content/main.js

```diff
@@ -87,7 +87,10 @@ var gMainPane = {
     setEventListener("localeSelect", "popuphiding", function () {
       gMainPane.updateLocale();
     });
-
+    setEventListener("localeSelect", "keypress", function (e) {
```

Comment:
Binds to the `keypress` event, this case when keys are pressed on the focused control